tinyDA/link.py

```diff
@@ -46,3 +46,12 @@ def __init__(self, parameters, prior, model_output, likelihood, qoi=None):
 
         # compute the (unnormalised) posterior.
         self.posterior = self.prior + self.likelihood
+
+
+class RandomProposal:
+    """Separate object for each sample on the fine level, that points to the 
+    proposal that was promoted by the subchain.
+    
+    Attributes
+    ----------
+    """
```

Comment:
This doesn't seem to do anything.

Reply:
Oh yes, this seems to be a remnant of an idea that I did not go through with. I removed it.